sale_product_pack/__manifest__.py

```diff
@@ -2,7 +2,7 @@
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
 {
     "name": "Sale Product Pack",
-    "version": "17.0.2.0.1",
+    "version": "17.0.2.0.2",
```

Comment:
```suggestion
    "version": "17.0.2.0.2",
```
This change is not necessary.

Reply:
hum, i'm changing some code so I bump the version.

maybe as a refactor it's not a bugfix and you want to change it to `17.0.2.1.0` ?

Reply:
When the bot is merged it will make this change automatically. 
You only need to change the version on migrations and/or if migration scripts are needed.

Reply:
ah ok nice. fixing this

Reply:
@damdam-s